pwa/src/scss/PatientInfoForm.scss

```diff
@@ -0,0 +1,17 @@
+#patient-info-wrapper {
```

Comment:
Do we have some standard for the project on using `id` vs `class` css selectors?

Reply:
Currently Meghan and I have naturally been working under the assumption that:

If it's a style being used for one component (unique) we apply using it's ID.
If it's reused within scope (contained within an id like the case of the wrapper, it should be a class)

We can make sure to document that definitively in #45 

If it's documented / discussed further in that ticket, is that enough to resolve this review comment?

Reply:
That sounds fine. If we have time, I would also like to put together a little development guide at the end of this semester w/ general practices for dev (styling, frameworks, etc...). But for now, I'm good with noting it in #45